src/Jank/TermManagement/TermList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Term from './Term';
+import useApi from '../../hooks/useApi';
+import { AddButton, AddLabel, TermInput } from './ui';
+
+type Terms = {
+  _id: string,
+  value: string
+}
+
+const TermList = () => {
+  const [error, terms] = useApi<Terms[]>('/jank/terms');
+
+  if (error) {
+    return <span>Begriffe konnten leider nicht geladen werden</span>;
+  }
+  return (
+    <>
+      <h1>Begriffe verwalten</h1>
+      <form>
+        <AddLabel>
+          Begriff hinzufügen:
+          <TermInput type="text" name="word" />
+        </AddLabel>
+        <AddButton name="Submit">+</AddButton>
```

Comment:
The button here usually doesn't need a `name`, since it doesn't carry any data. Maybe this was meant to be `type="submit"`? That would be correct - although not technically necessary, since its the default. I wouldn't mind being explicit here though.